protostar/cheatable_starknet/callable_hint_locals/print_hint_local.py

```diff
@@ -0,0 +1,30 @@
+from typing import Callable
+
+from protostar.starknet import CheatcodeException
+from protostar.cheatable_starknet.controllers.contracts import (
+    ContractsCheaterException,
+    ContractsController,
+)
+from protostar.starknet.data_transformer import CairoData
+
+from .callable_hint_local import CallableHintLocal
+
+
+class PrintHintLocal(CallableHintLocal):
+    def __init__(self, controller: ContractsController):
+        self._controller = controller
+
+    @property
+    def name(self) -> str:
+        return "protostar_print"
+
+    def _build(
+        self,
+    ) -> Callable[[CairoData], None]:
+        return self.protostar_print
+
+    def protostar_print(self, data: CairoData) -> None:
+        try:
+            self._controller.protostar_print(data=data)
+        except ContractsCheaterException as exc:
```

Comment:
what exceptions do we expect to happen here?

Reply:
If we expect an exception in any scenario I would like to see a test including it

Reply:
[address feedback](https://github.com/software-mansion/protostar/pull/1803/commits/c5e924eec416744cb73082e5f414b88d3e395c5e)